CMakeLists.txt

```diff
@@ -121,78 +76,6 @@ add_service_files(
         LIDoutputstateSrv.srv
 )
 
-if(ENABLE_EMULATOR)
-    # emulator messages
-    add_message_files(
-        DIRECTORY test/emulator/msg
-        FILES
-        SickLocColaTelegramMsg.msg
-        SickLocDiagnosticMsg.msg
-        SickLocResultPortCrcMsg.msg
-        SickLocResultPortHeaderMsg.msg
-        SickLocResultPortPayloadMsg.msg
-        SickLocResultPortTelegramMsg.msg
-        SickLocResultPortTestcaseMsg.msg
-    )
-
-    # emulator services
-    add_service_files(
-        DIRECTORY test/emulator/srv
-        FILES
-        SickLocColaTelegramSrv.srv
-        SickLocRequestTimestampSrv.srv
-        SickLocSetResultModeSrv.srv
-        SickLocSetResultPoseIntervalSrv.srv
-        SickLocIsSystemReadySrv.srv
-        SickLocSetPoseSrv.srv
-        SickLocSetResultPortSrv.srv
-        SickLocStartLocalizingSrv.srv
-        SickLocStopSrv.srv
-        SickLocRequestResultDataSrv.srv
-        SickLocSetResultEndiannessSrv.srv
-        SickLocSetResultPoseEnabledSrv.srv
-        SickLocStateSrv.srv
-        SickLocTimeSyncSrv.srv
-        SickDevGetLidarConfigSrv.srv
-        SickDevGetLidarIdentSrv.srv
-        SickDevGetLidarStateSrv.srv
-        SickDevSetLidarConfigSrv.srv
-        SickGetSoftwareVersionSrv.srv
-        SickLocAutoStartActiveSrv.srv
-        SickLocAutoStartSavePoseIntervalSrv.srv
-        SickLocAutoStartSavePoseSrv.srv
-        SickLocForceUpdateSrv.srv
-        SickLocInitializePoseSrv.srv
-        SickLocInitialPoseSrv.srv
-        SickLocMapSrv.srv
-        SickLocMapStateSrv.srv
-        SickLocOdometryActiveSrv.srv
-        SickLocOdometryPortSrv.srv
-        SickLocOdometryRestrictYMotionSrv.srv
-        SickLocReflectorsForSupportActiveSrv.srv
-        SickLocResultEndiannessSrv.srv
-        SickLocResultModeSrv.srv
-        SickLocResultPortSrv.srv
```

Comment:
the srv's are removed here but present in the file tree; same holds for the messages